uwblueprint/supportive-housing
# Add Resident Form

## GitHub Issue link

[Add Resident Form](https://github.com/uwblueprint/supportive-housing/issues/91)



## Implementation description
* Created `CreateResident` component for the input form
* Added new `Resident` type for inputted information
* Created post API in `CommonAPIClient` 



## Steps to test
1. Verify input validation occurs when information is missing from the form
2. Verify information that is submitted creates a new resident and gets added to the database



## What should reviewers focus on?
* Does a new APIClient file need to be created?

Screenshot:
![image](https://github.com/uwblueprint/supportive-housing/assets/43042601/5b51d687-957a-45f4-a98d-b20a94c3ba3e)

## Checklist
